### DISCUSSION TITLE
PCL2启动器是否会有64位的计划

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#367] [提问] [answered]
如题，附图
![ProcessPCL32Bit](https://user-images.githubusercontent.com/117887221/201335309-58e1183e-97ef-4505-b1d3-1ef9e70cc86e.jpg)

**Comment by Pigeon0v0** [comment]
64 位没必要吧

**Reply by GTEMYee** [reply]
但是BakaXL是64位
![ProcessInfoBakaXL64Bit](https://user-images.githubusercontent.com/117887221/201512407-7b03aece-7558-40e4-a3c2-83d42605aa69.jpg)
或许让龙腾猫跃来解答一下?🤔

**Comment by LTCatt** [comment]
![image](https://user-images.githubusercontent.com/40348278/201712038-067b0345-b45f-4da6-a8a3-4ff0c04b8901.png)

PCL2 应当会优先以 64 位运行，不知道你那是咋回事？
其他人也是这样吗？如果是的话可以开个 issue？

**Reply by GTEMYee** [reply]
64位会不会跟预览版有关系
那正式版会不会是只有32位🤔

**Comment by HarukaB** [comment]
PCL2一直是64位的，
![image](https://user-images.githubusercontent.com/100848181/201810920-80b6d0f0-7e4e-4b3a-b6fc-69c0aa5e93db.png)
这应该是你自己的问题

**Reply by GTEMYee** [reply]
可能是我用的正式版吧
![ProgramPCLInfo](https://user-images.githubusercontent.com/117887221/201863465-84d2592b-a120-49b5-9686-0113dfc86bfa.jpg)

**Comment by LTCatt** [comment, marked as answer]
确实，看起来正式版会优先以 32 位运行……你交个 issue 吧
